.github/CODEOWNERS

```
@@ -10,3 +10,4 @@
 /Modules/ITS/      @iravasen @IsakovAD
 /Modules/TRD/      @martenole @bazinski
 /Modules/FOCAL/    @mfasDa @fjonasALICE
+/Modules/FIT/      @andreasmolander, @afurs, @jotwinow
```

```suggestion
/Modules/FIT/      @andreasmolander @afurs @jotwinow
```
Shouldn't it be without commas?

---

correct !